fe/lib/.npmignore

```diff
@@ -0,0 +1,9 @@
+/storybook/**/*
```

Comment:
Is this just for package size reasons?

I was reading the Storybook v6 documentation and I'm starting to see a method in their madness. We could *almost* automatically lint links to our Storybook from the Developer Site without taking any Storybook-specific deps. We'd just need to collect all the named exports from the `.stories.tsx` files.

`withBraidProvider` takes a dep on knobs but maybe we could move that to a global decorator so it's not explicitly imported.

Reply:
Yep, and it's not huge savings. I'll re-include the stories.